src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/security/AuthorisedCaseSearchOperation.java

```diff
@@ -74,7 +80,13 @@ public CaseSearchResult execute(CrossCaseTypeSearchRequest searchRequest) {
 
     private CrossCaseTypeSearchRequest createAuthorisedSearchRequest(List<CaseType> authorisedCaseTypes, CrossCaseTypeSearchRequest originalSearchRequest) {
         List<String> authorisedCaseTypeIds = authorisedCaseTypes.stream().map(CaseType::getId).collect(Collectors.toList());
-        return new CrossCaseTypeSearchRequest(authorisedCaseTypeIds, originalSearchRequest.getSearchRequestJsonNode());
+
+        return new CrossCaseTypeSearchRequest.Builder()
+            .withCaseTypes(authorisedCaseTypeIds)
+            .withSearchRequest(originalSearchRequest.getSearchRequestJsonNode())
+            .withMultiCaseTypeSearch(originalSearchRequest.isMultiCaseTypeSearch())
+            .withSourceFilterAliasFields(originalSearchRequest.getSourceFilterAliasFields())
+            .build();
     }
 
     private CaseSearchResult searchCasesAndFilterFieldsByAccess(List<CaseType> authorisedCaseTypes, CrossCaseTypeSearchRequest authorisedSearchRequest) {
```

Comment:
can we get rid of the first parameter in this method? We are already passing the authorisedSearchRequest and the list of authorisedCaseTypes can be retrieved from there, if I'm not missing anything

Reply:
same for the method `filterFieldsByAccess`